plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1242,6 +1242,13 @@ public void setViewComment(ConnectorSession session, SchemaTableName viewName, O
         trinoViewHiveMetastore.updateViewComment(session, viewName, comment);
     }
 
+    @Override
+    public void setViewColumnComment(ConnectorSession session, SchemaTableName viewName, String columnName,
+            Optional<String> comment)
```

Comment:
Please unwarp.

Reply:
Do you means let the parameter `Optional<String> comment` of `io.trino.plugin.hive.TrinoViewHiveMetastore#updateViewColumnComment` should be changed to `String comment`? I just follow the `updateViewComment` method and the constructor for `ConnectorViewDefinition.ViewColumn` is also expects a `Optional<String>`.

Reply:
I mean putting all arguments in a single line.